src/services/GraphQLQueryGenerator.js

```diff
@@ -0,0 +1,40 @@
+class GraphQLRG {
+    static generateBody(_method, _function, _queryData, _variables) {
```

Comment:
Method is too big and very hard to follow.

Reply:
small brain, child

Reply:
report for bully